package.json

```diff
@@ -31,8 +31,8 @@
     "@nestjs/terminus": "^8.0.1",
     "@nestjs/websockets": "^8.0.6",
     "@types/cache-manager": "^3.4.2",
-    "cache-manager": "^4.1.0",
-    "cache-manager-redis-store": "^3.0.1",
+    "cache-manager": "^3.6.0",
+    "cache-manager-redis-store": "^2.0.0",
```

Comment:
This was downgraded due to compatibility issues with redis store.
Look [here](https://stackoverflow.com/questions/74621340/i-get-a-types-error-when-registering-redis-with-nestjs-using-cachemodule-registe).